.github/workflows/checks.yml

```diff
@@ -0,0 +1,149 @@
+name: Checks
+
+on:
+  workflow_call:
+    secrets:
+      ALTERNATIVE_GITHUB_TOKEN:
+        required: false
+
+jobs:
+
+  Version-Check:
+    name: Version
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: SCM Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup Python & Poetry Environment
+        uses: exasol/python-toolbox/.github/actions/python-environment@0.15.0
+
+      - name: Check Version(s)
+        run: poetry run version-check "conda_vulnerability_scanner/version.py"
+
+  Documentation:
+    name: Docs
+    needs: [ Version-Check ]
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: SCM Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Python & Poetry Environment
+        uses: exasol/python-toolbox/.github/actions/python-environment@0.15.0
+
+      - name: Build Documentation
+        run: |
+          poetry run python -m nox -s build-docs
+
+  #Lint:
+  #  name: Linting (Python-${{ matrix.python-version }})
+  #  needs: [ Version-Check ]
+  #  runs-on: ubuntu-latest
+  #  strategy:
+  #    fail-fast: false
+  #    matrix:
+  #      python-version: ["3.8", "3.9", "3.10", "3.11"]
+
+  #  steps:
+  #    - name: SCM Checkout
+  #      uses: actions/checkout@v4
+
+  #    - name: Setup Python & Poetry Environment
+  #      uses: exasol/python-toolbox/.github/actions/python-environment@0.15.0
+  #      with:
+  #        python-version: ${{ matrix.python-version }}
+
+  #    - name: Run lint
+  #      run: poetry run nox -s lint
+
+  #    - name: Upload Artifacts
+  #      uses: actions/upload-artifact@v4.4.0
+  #      with:
+  #        name: lint-python${{ matrix.python-version }}
+  #        path: .lint.txt
+  #        include-hidden-files: true
+
+  #Type-Check:
+  #  name: Type Checking (Python-${{ matrix.python-version }})
+  #  needs: [ Version-Check ]
+  #  runs-on: ubuntu-latest
+  #  strategy:
+  #    fail-fast: false
+  #    matrix:
+  #      python-version: ["3.8", "3.9", "3.10", "3.11"]
+
+  #  steps:
+  #    - name: SCM Checkout
+  #      uses: actions/checkout@v4
+
+  #    - name: Setup Python & Poetry Environment
+  #      uses: exasol/python-toolbox/.github/actions/python-environment@0.15.0
+  #      with:
+  #        python-version: ${{ matrix.python-version }}
+
+  #    - name: Run type-check
+  #      run: poetry run nox -s type-check
```

Comment:
Just remove those steps, but create an issue for introducing them inf the future.
```suggestion
```